package.json

```diff
@@ -1,16 +1,28 @@
 {
-  "name": "dependents-testing",
+  "name": "dependents",
   "version": "0.1.0",
-  "description": "Tool to test a modules dependents",
-  "main": "index.js",
+  "description": "Tool to get a modules dependents",
   "bin": "bin/dependents.js",
+  "scripts": {
+    "test": "standard && mocha test/index.js",
+    "coverage": "nyc --reporter=html --reporter=text --reporter=text-summary npm test"
+  },
   "author": "Andrew Hughes",
   "license": "Apache-2.0",
-  "repository": "andrewhughes101/dependents-testing",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:pkgjs/dependents.git"
+  },
   "dependencies": {
     "@octokit/graphql": "^4.3.1",
```

Comment:
Is the GitHub GraphQL endpoint still used? I couldn't spot it from my quick skim.

Reply:
Yep used in `lib/github.js`